docs/products/mysql/concepts/high-availability-mysql.md

```diff
@@ -0,0 +1,112 @@
+---
+title: High availability of Aiven for MySQL®
```

Comment:
I rethought about this after doing my previous review.  In this doc, there are many references to backups and restores.  Maybe all the details about these should be put in a Backups and Restores page and a link to this new page should be put in here.  This might simplify the structure of this doc.

Reply:
This is a good idea indeed. We might follow up on it in the future. For now, the bandwidth allows one new document only :-) Feel free to created a [DOC Jira](https://aiven.atlassian.net/jira/software/c/projects/DOC/boards/155) ticket for this future improvement. Thanks!

Reply:
DOC Jira done: https://aiven.atlassian.net/browse/DOC-858